### PR TITLE
First attempt at documenting serverIdleTimeout

### PR DESCRIPTION
**Pull request by lervag**
This is not ready for merge. There are a few questions:

1. Is this the correct place to add this type of reference?
2. Is it the correct way to document it?
3. Should I format it differently?

I would not mind completing this and perhaps try and include a few more settings when I'm at it.

Please see discussion at https://github.com/sbt/sbt/issues/7631.

### CODE REVIEW

**Comment by eed3si9n** [comment]
Yea. I guess we don't yet have user-facing sbt server page, but I think this is fine for now.

File: src/reference/02-DetailTopics/01-Using-sbt/07-sbt-server.md [hunk 1]
```diff
@@ -332,3 +332,16 @@ This will respond back with the result of the action.
   [lsp_initialize]: https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#initialize
   [lsp_initialized]: https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#initialized
   [lsp_publishdiagnosticsparams]: https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#publishdiagnostics-notification
+
+### Configuration
```
Comment (eed3si9n) [inline src/reference/02-DetailTopics/01-Using-sbt/07-sbt-server.md]: I think this should go immediately after the intro, like at line 17.

**Comment by eed3si9n** [comment]
Also could you target 1.x branch for sbt 1.x documentation plz?

**Comment by lervag** [comment]
I believe I've made the suggested changes.

Should we include more options here before considering a merge? If so, are there any particular ones that seem more important?

**Comment by eed3si9n** [comment]
> Should we include more options here before considering a merge? If so, are there any particular ones that seem more important?

I'd say ship first and we can add more later.

**Pull request merged**